### ISSUE TITLE
Render Error

### ISSUE THREAD

**Issue by Konradrundfunk** [IzakMarais/reporter#247]
Hi,
I have seen so many people complaining about this error but couldn't find a proper solution.
```
2020/08/19 19:44:10 Connecting to dashboard at http://192.168.178.38:3000/api/dashboards/uid/UfVUm4RRz
2020/08/19 19:44:11 Populated dashboard datastructure: {Title:Temperatur Description: VariableValues: Rows:[] Panels:[{Id:2 Type:graph Title:Temperatur Kühlschrank GridPos:{H:13 W:24 X:0 Y:0}}]}
2020/08/19 19:44:11 Downloading image  2 http://192.168.178.38:3000/render/d-solo/UfVUm4RRz/_?from=now-1y&height=500&panelId=2&theme=light&to=now&width=1000
2020/08/19 19:44:11 Error obtaining render for panel {Id:2 Type:graph Title:Temperatur Kühlschrank GridPos:{H:13 W:24 X:0 Y:0}}, Status: 500, Retrying after 10s...
2020/08/19 19:44:22 Error obtaining render for panel {Id:2 Type:graph Title:Temperatur Kühlschrank GridPos:{H:13 W:24 X:0 Y:0}}, Status: 500, Retrying after 20s...
2020/08/19 19:44:42 Error obtaining render: <!DOCTYPE html>
```
I think the problem here is the api key it's not passed. Or does grafana save it while the request?
Or do I need a special grafic engine for that?
I am using a Raspberry Pi 3 B 
on Grafana Version 6.4.4
newest Grafana-reporter

**Comment by vincedjango** [comment]
Hi @Konradrundfunk ,
I solved it. 
The first thing to understand is that the error does not come directly from this plugin **Grafana Reporter,** it comes from a plugin used by **Grafana Reporter**. The source of the issue is in the grafana official plugin, [Grafana Image Renderer](https://grafana.com/grafana/plugins/grafana-image-renderer)


I'm using Grafana 7, with Centos7, and everything was working fine, until I decided to delete Google Chrome, and then I got this error.  Even after reinstalling chrome, the rendering image would not go away.

1. I looked at the log in /var/log/grafana/grafana.log and found this:
`error="Error: Failed to launch chrome!\n/var/lib/grafana/plugins/grafana-image-renderer/chrome-linux/chrome: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory\n\n\nTROUBLESHOOTING: `
Install libXss.so.1 or Chrome does not fix the issue.

2. **Solution:**

- You do not have to install Google Chrome, it can be installed, it does not cause any harm, its just not necessary for this error.

- InstaIl `sudo yum install chromium` for Centos or `sudo apt install chromium` for ubuntu
